src/nodes/packet_relay_node.c

```diff
@@ -61,9 +86,12 @@ static __rte_always_inline rte_edge_t get_next_index(__rte_unused struct rte_nod
 		return PACKET_RELAY_NEXT_IPIP_ENCAP;
 	}
 
-	if (df->l4_type == DP_IP_PROTO_ICMP)
+	if (df->l4_type == IPPROTO_ICMP)
```

Comment:
I'm all for this change (again something on my TODO), but it should be then changed everywhere for consistency.

Reply:
Yeah I agree but current code base has already mixed usage so this one is not making it worse so let's clean it up in a follow-up refactoring PR.